types/algoliasearch/index.d.ts

```diff
@@ -24,25 +25,25 @@ declare namespace algoliasearch {
      * Query on multiple index
      * https://github.com/algolia/algoliasearch-client-js#multiple-queries---multiplequeries
      */
-    search(
+    search<T=any>(
```

Comment:
Can you make this `T` into `Hit` to make it clearer what it is?

Reply:
It seems that the notation `T` is something pretty common in definition file to indicate that it's a generic vs a regular type. We can use `THits` to avoid confusion.

Reply:
I agree with @samouss. Generally I'm fine with either `T` or `THit`.